csharp/src/Ice/StringUtil.cs

```diff
@@ -9,21 +9,21 @@
 
 namespace ZeroC.Ice
 {
-    internal static class StringUtil
+    public static class StringUtil
     {
         //
         // Return the index of the first character in str to
         // appear in match, starting from 0. Returns -1 if none is
         // found.
         //
-        public static int FindFirstOf(string str, string match) => FindFirstOf(str, match, 0);
+        internal static int FindFirstOf(string str, string match) => FindFirstOf(str, match, 0);
 
         //
         // Return the index of the first character in str to
         // appear in match, starting from start. Returns -1 if none is
         // found.
         //
-        public static int FindFirstOf(string str, string match, int start)
+        internal static int FindFirstOf(string str, string match, int start)
```

Comment:
I known it's not part of this PR but are these really useful. Doesn't C# have an `indexOf` method on `string`.

Reply:
there is `IndexOfAny` but it uses a char array instead of a string for the match

https://docs.microsoft.com/en-us/dotnet/api/system.string.indexofany?view=netcore-3.1#System_String_IndexOfAny_System_Char___